src/modules/comment_api.js

```diff
@@ -0,0 +1,34 @@
+const commentApi = (e) => {
+  const involmentAppID = 'hrqIdiXTh94rmLQMrXcG';
+  const index = Number(e.target.dataset.id);
+  const date = new Date();
+  const month = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
+  const num = month[date.getMonth()];
+  const actualDate = `${date.getFullYear()}-${num}-${date.getDate()}`;
+  const name = e.target.parentElement.children[0].children[1].value;
+  const comment = e.target.parentElement.children[1].children[0].value;
+  if (name.length < 1 && comment.length < 1) {
+    return false;
+  }
+  fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${involmentAppID}/comments`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: index,
+      comment,
+      username: name,
+    }),
```

Comment:
- [OPTIONAL] Great job the user can post a comment  👍🏻  however 🤔  I'm getting this error when creating a new comment, Please correct this. 💯 👍🏻  

![image](https://github.com/rachaouldbabaali/JavaScript-capstone-/assets/79058364/1241f715-a976-47c6-befa-117a41bae699)